src/tools/dev/scripts/bv_support/bv_icet.sh

```diff
@@ -92,78 +92,16 @@ function apply_icet_patch
 
 function build_icet
 {
-    PAR_INCLUDE_STRING=""
-    if [[ "$PAR_INCLUDE" != "" ]] ; then
-        PAR_INCLUDE_STRING=$PAR_INCLUDE
-    fi
-
-    if [[ "$PAR_COMPILER" != "" ]] ; then
-        if [[ "$OPSYS" == "Darwin" && "$PAR_COMPILER" == "/usr/bin/mpicc" ]]; then
-            PAR_INCLUDE_STRING="-I/usr/include/"
-        elif [[ "$OPSYS" == "Linux" && "$PAR_COMPILER" == "mpixlc" ]]; then
-            PAR_INCLUDE_STRING=`$PAR_COMPILER -show`
-        else
-            if [[ -z "$PAR_INCLUDE_STRING" ]]; then
-                PAR_INCLUDE_STRING=`$PAR_COMPILER --showme:compile`
-                if [[ $? != 0 ]] ; then
-                    PAR_INCLUDE_STRING=`$PAR_COMPILER -show`
-                fi
-            fi
-        fi
-    fi
-
-    if [[ "$PAR_INCLUDE_STRING" == "" ]] ; then
-        warn "You must set either the PAR_COMPILER or PAR_INCLUDE environment variable to build Ice-T."
+    # We need to set MPI_COMPILER for CMake. We can get that from MPICH
+    # or from PAR_COMPILER. Note that DO_MPICH causes PAR_COMPILER to be
+    # set, so testing for that also catches using MPICH.
+    if [[ "$PAR_COMPILER" == "" ]] ; then
+        warn "You must use MPICH or set either the PAR_COMPILER environment variable to build Ice-T."
```

Comment:
I don't think 'use MPICH' is clear enough, they need to add --mpich to the build_visit command line correct?
Also, I believe 'either is in the wrong location. 
Perhaps:

To build Ice-T, you must either use MPICH (add --mpich) or set the PAR_COMPILER environment variable.

Reply:
I changed the message as you suggested. I also changed all the references of Ice-T to IceT, since they don't use the dash on their website.